features/file.yml

```diff
@@ -1,7 +1,12 @@
-draft_date: 2024-10-21
-name: File API
-description: TODO
+name: File
+description: The `File` API provides information about files and allows JavaScript in a web page to access their content.
```

Comment:
Should this also mention the ability to create files as well? I've used this for creating CSVs to be downloaded or images for display/download, for example.

Reply:
yes, thanks! updated